server/sonar-web/src/main/js/apps/component-measures/drilldown/TreeMapView.js

```diff
@@ -102,15 +102,15 @@ export default class TreeMapView extends React.PureComponent {
   getLevelColorScale = () =>
     scaleOrdinal().domain(['ERROR', 'WARN', 'OK', 'NONE']).range(LEVEL_COLORS);
 
-  getPercentColorScale = (metric: Metric) => {
+  getPercentColorScale = (metric /*: Metric */) => {
     const color = scaleLinear().domain([0, 25, 50, 75, 100]);
     color.range(metric.direction === 1 ? COLORS.reverse() : COLORS);
     return color;
   };
 
   getRatingColorScale = () => scaleLinear().domain([1, 2, 3, 4, 5]).range(COLORS);
 
-  getColorScale = (metric: Metric) => {
+  getColorScale = (metric /*: Metric */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)